it/uk/gov/hmrc/tai/integration/cache/connectors/CacheConnectorItSpec.scala

```diff
@@ -119,14 +126,14 @@ class CacheConnectorItSpec extends TaiBaseSpec("CacheConnectorItSpec") with Mong
 
       "return the data from cache" when {
         "Nil is saved in cache" in {
-          val data = Await.result(sut.createOrUpdate[Seq[SessionData]](cacheId, Nil), atMost)
+          Await.result(sut.createOrUpdate[Seq[SessionData]](cacheId, Nil), atMost)
```

Comment:
Should we use futureValue here? Or wait until we start the Bootstrap/Play upgrade?